app/assets/javascripts/discourse/controllers/bookmark.js

```diff
@@ -71,9 +111,44 @@ export default Controller.extend(ModalFunctionality, {
     }
   },
 
+  bindKeyboardShortcuts() {
+    if (!this.mouseTrap) {
+      this.mouseTrap = Mousetrap(
+        document.querySelector("#discourse-modal.bookmark-with-reminder")
+      );
+    }
+
+    GLOBAL_SHORTCUTS_TO_PAUSE.forEach(combo => Mousetrap.unbind(combo));
+
+    Object.keys(bindings).forEach(key => {
+      this.mouseTrap.bind(key, e => {
```

Comment:
this is personal preference and very minor nitpick but I prefer `event` over `e` as it's way easier to grep for it in the codebase.